requirements/local.txt

```diff
@@ -0,0 +1,14 @@
+# Packages required for local development
+-r test.txt
+
+backports.shutil-get-terminal-size==1.0.0
```

Comment:
Same question as `base.txt`. If we aren't actively using any of these, let's kick them out to avoid future confusion (and false-alerts from Gemnasium).